src/server/web_server.js

```diff
@@ -441,6 +439,55 @@ function getVersion(route) {
         });
 }
 
+// An oauth authorize endpoint that forwards to the OAuth authorization server.
```

Comment:
Uh, why do you need this?

Reply:
As a design decision, I aspired to keep most of the knowledge about the OAuth server and the plumbing related to the OAuth "handshake" (the grant code flow) inside our server which holds all the knowledge regarding the OAuth endpoint and protocol mostly hidden from the Management console.

In order to achieve this, I defined a route on our web server to act as a forwarder to the OAuth server so the Management console only has to redirect to this route without any knowledge of the auth server location or the parameters that have to be sent as part of the OAuth grant code flow.

Reply:
Okay.